pkg/util/fedinformer/typedmanager/multi-cluster-manager.go

```diff
@@ -20,9 +21,9 @@ func init() {
 }
 
 // GetInstance returns a shared MultiClusterInformerManager instance.
-func GetInstance() MultiClusterInformerManager {
+func GetInstance(transformFuncs map[schema.GroupVersionResource]cache.TransformFunc) MultiClusterInformerManager {
```

Comment:
The `GetInstance` actually just initializes at the first time it is called, which means only the first input takes effect, but the parameters by following calls will be ignored.

That's probably unacceptable.